internal/cmd/commands/database/funcs.go

```diff
@@ -25,6 +25,8 @@ func migrateDatabase(ctx context.Context, ui cli.Ui, dialect, u string, initiali
 		ui.Error(fmt.Errorf("Error establishing db connection: %w", err).Error())
 		return noop, 2
 	}
+	dBase.SetMaxOpenConns(maxOpenConns)
+	defer dBase.Close()
```

Comment:
Hm, I'm being told this may be intentionally kept open, so this may be incorrect.

Reply:
Remove it